go/test/endtoend/vtgate/queries/misc/misc_test.go

```diff
@@ -519,3 +519,24 @@ func TestTimeZones(t *testing.T) {
 		})
 	}
 }
+
+// TestSemiJoin tests that the semi join works as intended.
+func TestSemiJoin(t *testing.T) {
+	utils.SkipIfBinaryIsBelowVersion(t, 22, "vtgate")
```

Comment:
do we need this now?

Reply:
Yes, the test is still flaky for older releases, although it fails very infrequently.

Reply:
we do not run v22 test on v21 VTGate in upgrade/downgrade test.
To really skip this test, this should be in `release-21` branch than on `main`

We have already planned the backport. So that will not be an issue.

Reply:
I removed the line.